docs/boards/Board - CrazyBeeF3FR.md

```diff
@@ -71,7 +71,11 @@ https://www.banggood.com/Racerstar-Crazybee-F3-Flight-Controller-4-IN-1-5A-1S-Bl
 
 Specific information for the factory supplied Betaflight 3.3.0 version:
 
-- The DSHOT beeper function does not work.
+- To activate DSHOT beeper, you need to go in CLI and write 
+
+	*set beeper_dshot_beacon_tone=1*<br/>*save*<br/>
```

Comment:
While technically true, this will actually cause a FrSky SPI flight controller to lose the RX connection when arming in 3.3, so giving this advice here is not helpful.

Reply:
ok. thank you for the clarification. I am actually flying that CrazyBeeF3 board on my brushless tinywhoop and I can use DSHOT beacon and still fly after arming. I never notice any problems. I arm, fly, use beacon, arm, fly etc.

Reply:
That's with 3.3?

Reply:
ok I just went and tested something. When I activate the DSHOT beacon, the motors beep BUT I do lose telemetry, my taranis start saying "lost telemetry" and when I arm again to fly, it stop saying lost telemetry.